processor/src/networks/monero.rs

```diff
@@ -376,6 +376,12 @@ impl Monero {
       })
       .collect::<Vec<_>>();
 
+    let change = Change::fingerprintable(if change.is_some() {
+      Some(change.clone().unwrap().into())
```

Comment:
`if let Some(change) = change {}`, to avoid the unwrap

Reply:
Did that suggestion, then the compiler warned it's equivalent to `Option::map` and  suggested `change.as_ref().map(|change| change.clone().into())`. See the latest. I was wondering what the cleanest way to write this was, cool.